### PR TITLE
Include the BBB Mock container

### PR DESCRIPTION
**Pull request by andrewnicols**

### CODE REVIEW

File: config.docker-template.php [hunk 1]
```diff
@@ -97,4 +97,13 @@
     define('TEST_ENROL_LDAP_DOMAIN', 'ou=Users,dc=openstack,dc=org');
 }
 
+if (getenv('BBBMOCK')) {
```
Comment (stronk7) [inline config.docker-template.php:100]:
```suggestion
if (getenv('MOODLE_DOCKER_BBB_MOCK')) {
```

I'd suggest to use the very same name and then add it to the `bbb-mock.yml`file. See above.

File: bbb-mock.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+version: "2"
+services:
+  bbbmock:
```
Comment (stronk7) [inline bbb-mock.yml:3]:
```suggestion
  webserver:
    environment:
      MOODLE_DOCKER_BBB_MOCK: "true"
  bbbmock:
```

Let's declare the env variable here, so `config.php` is able to get it.

**Review by stronk7 — CHANGES_REQUESTED** [review]
Hi,

it looks nearly ok. But the `BBBMOCK` used in the `config.php` template does not exist.

Because of that, and following a similar path used for other env variables, I've suggested to create the env variable (with the same name) in the new yml file and then just read it.

File: config.docker-template.php (hunk 1 above)
Comment (andrewnicols) [reply config.docker-template.php:100]: Agree - thanks.

**Comment by mattporritt** [comment]
Suggested changes have been made.
I've also tested changes. Merged

**Pull request merged**